Tests/Unit/Models/RemoteToLocalCategoryTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ShopwarePlugins\Connect\Tests\Unit\Components;
+
+use Shopware\CustomModels\Connect\RemoteCategory;
+use Shopware\CustomModels\Connect\RemoteToLocalCategory;
+use Shopware\Models\Category\Category;
+
+class RemoteToLocalCategoryTest extends \PHPUnit_Framework_TestCase
```

Comment:
same here. I think will be better if we can make assertions in some integration test.

Reply:
we have integration test for most of it

Reply:
problem is currently that we try to use less and less doctrine entities because of performance so some of this isn't used anymore